### DISCUSSION TITLE
Problemas con bootcamps

### DISCUSSION THREAD

**Discussion by Aresdgi** [ivansaul/codigo_facilito_downloader#41] [General]
Al intentar descargarte una clase de un bootcamp, tienes el problema que normalmente se divide en varias clases, al requerirte iniciar sesion para verlo, solo se descarga la primera que suele ser una introducción. una lastima

**Comment by ivansaul** [comment]
Hola, @Aresdgi. El cli no tiene soporte para los bootcamps. En ese caso tienes que descargar uno a uno. En la documentación se muestran algunos [ejemplos](https://github.com/ivansaul/codigo_facilito_downloader?tab=readme-ov-file#instrucciones)
